svd-rs/src/cluster.rs

```diff
@@ -70,9 +81,9 @@ mod ser_de {
         {
             let ClusterArray { dim, info } = ClusterArray::deserialize(deserializer)?;
             if let Some(dim) = dim {
-                Ok(Cluster::Array(info, dim))
+                Ok(Self::Array(info, dim))
```

Comment:
Why aren't these using the new functions?

Reply:
done